src/LudoGame/GameEngine/Class1.cs

```diff
@@ -52,12 +52,12 @@ public class Piece
         public int correctionFactor { get; set; }
```

Comment:
Can you please explain what you mean by correctionFactor

Reply:
Varje pjäs har en "lokal" position (antal steg som pjäsen har gått) och en "global" position (vilken ruta på brädan pjäsen står på). De första fyra pjäserna som skapas för en spelare får correctionFactor = 0. Därefter ökas värdet med 10 för varje fjärde pjäs (0, 10, 20, 30). Om vi antar att en pjäs har correctionFactor = 20 och den har gått 16 steg kan vi räkna ut den absoluta positionen (var på brädan den är) genom att ta position + correctionFactor = 20 + 16 = 36. Den befinner sig alltså på ruta 36. Genom att räkna ut den absoluta/globala positionen kan vi jämföra två pjäser med varandra för att avgöra om de står på samma ruta.

Sammanfattat är correctionFactor en "konstant" som adderas med pjäsens lokala position för att få fram dess globala position.

Reply:
Skulle vara bra om denna beskrivning på något sätt är att finna i eran docs-mapp.

Reply:
Resolved.